.github/Style Guide.md

```diff
@@ -31,7 +31,11 @@ It will also contain guidance on aviation/SF-specific terms and how these ought
 
 ## Using this guide <a name="using"></a>
 
-This guide is intended to be guidelines, rather than firm and hard rules. Therefore, if it appears to be better style to break a 'rule', then this is encouraged. Indeed, it may be the case that a change is required in the guide, and if so, an issue should be opened.
+This guide is intended to be guidelines, rather than firm and hard rules. Therefore, if it appears to be better style to break a 'rule', then this is encouraged. The guide is also intended to be one that can improve and be flexible, so edits are encouraged in light of changing  best practice.

```

Comment:
```suggestion
This guide is intended to be guidelines, rather than firm and hard rules. Therefore, if it appears to be better style to break a 'rule', then this is encouraged. The guide is also intended to be one that can improve and be flexible, so edits are encouraged in light of changing best practice.

```